gradle/download-sentry-cli.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+cd $(dirname "$0")
+REPO=getsentry/sentry-cli
+VERSION=1.14.0
+PLATFORMS="Darwin-x86_64 Linux-i686 Linux-x86_64 Windows-i686"
+
```

Comment:
Probably want to `rm -f src/main/resources/bin/sentry-cli-*` here since they are ignored and who knows what artifacts people will collect over time.